store/src/java/com/zimbra/cs/service/mail/SendMsg.java

```diff
@@ -1,4 +1,4 @@
-	/*
+    /*
```

Comment:
Revert the change so that the file does not appear in code diff.

Reply:
Done.